src/workerd/api/url-standard.h

```diff
@@ -50,12 +50,12 @@ class URLSearchParams: public jsg::Object {
 
   static jsg::Ref<URLSearchParams> constructor(jsg::Optional<Initializer> init);
 
-  void append(kj::String name, kj::String value);
-  void delete_(jsg::Lock& js, kj::String name, jsg::Optional<kj::String> value);
-  kj::Maybe<kj::ArrayPtr<const char>> get(kj::String name);
-  kj::Array<kj::ArrayPtr<const char>> getAll(kj::String name);
-  bool has(jsg::Lock& js, kj::String name, jsg::Optional<kj::String> value);
-  void set(kj::String name, kj::String value);
+  void append(jsg::USVString name, jsg::USVString value);
+  void delete_(jsg::Lock& js, jsg::USVString name, jsg::Optional<jsg::USVString> value);
+  kj::Maybe<kj::ArrayPtr<const char>> get(jsg::USVString name);
+  kj::Array<kj::ArrayPtr<const char>> getAll(jsg::USVString name);
+  bool has(jsg::Lock& js, jsg::USVString name, jsg::Optional<jsg::USVString> value);
+  void set(jsg::USVString name, jsg::USVString value);
```

Comment:
Note: you'll likely need to also update the type generation code to handle `USVString`

Reply:
As we suspected, there aren't any changes to the generated types - it all comes out as "string".